sanic/request.py

```diff
@@ -80,6 +80,16 @@ def json(self):
 
         return self.parsed_json
 
+    def load_json(self, loads=json_loads):
```

Comment:
you should replace https://github.com/chiuczek/sanic/blob/a8f764c1612ab17eb99924ac2363c05110a1911d/sanic/request.py#L74-L81 with a call to this function

Reply:
of course I should, great suggestion, thanks